README.md

```diff
@@ -1,2 +1,70 @@
-# react-todo-app
-A simple To-Do app built with React (create app). This app allows users to add and delete tasks to organize their days.
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
```

Comment:
Please make your Readme more descriptive. This is the first stop that anyone who wants to see your project will land.  Giving a summary description ensures that users understand what it is about.